### PR TITLE
Address edge case of minVisible = 100%

### PR DESCRIPTION
**Pull request by jonkeller**
Spec allows you to give min, max visible percentage to trigger the event.
But it's defined as triggering on (min, max].
So if you want the event only when 100% of the creative is visible...you can't. If you say min=max=100 then the event will never fire, because the percent of the creative that is visible will never be > 100.

### CODE REVIEW

**Comment by lannka** [comment]
can we also decide what to do with min=0,max=0?

**Comment by jonkeller** [comment]
> "can we also decide what to do with min=0,max=0?"

Yes, absolutely.

**Comment by jonkeller** [comment]
I suppose we could do something like this in general:
if min == max: [min, max]
if min != max: (min, max]
Or is that too kludgey?

**Comment by lannka** [comment]
The 0 and 100 are edge cases can be handled specifically. I'm on board with your previous proposal:

`min=0,max=0` means to trigger when it's not visible
`min=100,max=100` means to trigger when it's fully visible
And these 2 usages need to be documented clearly.

**Comment by jonkeller** [comment]
Ok, SG.

**Comment by jonkeller** [comment]
@lannka I am being pulled in a different direction this week. What would you think about splitting this PR? We could keep 100% edge case here, and do 0% edge case in a different PR...

**Comment by lannka** [comment]
> @lannka I am being pulled in a different direction this week. What would you think about splitting this PR? We could keep 100% edge case here, and do 0% edge case in a different PR..

That sounds good to me. Please create a different issue to track. Also, do you want to add a small test for this new simple logic? The code looks good to me.

**Comment by jonkeller** [comment]
Sure, I can add a test. Thanks!

**Comment by jonkeller** [comment]
Created #12408

File: extensions/amp-analytics/0.1/test/test-visibility-model.js [hunk 1]
```diff
@@ -979,6 +979,22 @@ describes.sandboxed('VisibilityModel', {}, () => {
         totalVisibleTime: 1000,
       });
     });
+
+    it('should fire for 100 percent min and max visibility', () => {
+      const vh = new VisibilityModel({
+        continuousTimeMin: 1000,
```
Comment (lannka) [inline extensions/amp-analytics/0.1/test/test-visibility-model.js]: nit: can you remove the continuousTimeMin from the spec.
the test seems to be testing the timing, rather than the visible percentage.

Comment (jonkeller) [reply extensions/amp-analytics/0.1/test/test-visibility-model.js]: Good point. Done.

File: extensions/amp-analytics/0.1/test/test-visibility-model.js [hunk 2]
```diff
@@ -979,6 +979,18 @@ describes.sandboxed('VisibilityModel', {}, () => {
...
       });
     });
+
+    it('should fire for 100 percent min and max visibility', () => {
+      const vh = new VisibilityModel({
+        visiblePercentageMin: 100,
+        visiblePercentageMax: 100,
+      }, calcVisibility);
+      const eventSpy = vh.eventResolver_ = sandbox.spy();
+      visibility = 1.0;
```
Comment (lannka) [inline extensions/amp-analytics/0.1/test/test-visibility-model.js]: can you start with `visibility = 0.99` and check it's not called initially.

Comment (jonkeller) [reply extensions/amp-analytics/0.1/test/test-visibility-model.js]: Done
